StevesSixLeggedSprinters/BangazonAPI
# Kk customer

## Status
Choose:
**READY**

## Migrations
NO

## Description
Customer.cs and CustomerController.cs have been fully created.
Changes were also made to:
Bangazon.csproj
Controllers/ValuesController.cs
Data/BangazonAPIContext.cs
Program.cs
Startup.cs

We left some comments in Customer.cs.
We made a database file named bangazon.db

Each person will have to set up an environmental variable on their local.


## Todos


## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.
No migrations

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

git fetch --all
git checkout kk-customer
git pull origin kk-customer

(make sure to set environment variable locally)


## Impacted Areas in Application
List general components of the application that this PR will affect:

* This fixed all errors on the various files listed above
